docs/0.4/references/rust_api/index.md

```diff
@@ -0,0 +1,15 @@
+# Rust API documentation
+
+Reference guides for Splinter's Rust APIs:
+
+* [splinter API Reference](https://docs.rs/splinter/):
```

Comment:
These should specifically target the 0.4 versions (you can use `https://docs.rs/splinter/0.4` to point to the latest 0.4.x)

Reply:
Thanks!  Fixed.